TechnologyAssessmentApp/TechnologyAssessmentApp/Services/AppNetworkClient.swift

```diff
@@ -74,27 +77,29 @@ extension AppNetworkClient {
         let session = URLSession(configuration: config, delegate: self, delegateQueue: nil)
         // Ensure that an error is not present, otherwise, return the error
 
-        self.task = session.dataTask(with: newRequest, completionHandler: { (data, response, error) in
+        task = session.dataTask(with: newRequest, completionHandler: { (data, response, error) in
                 DispatchQueue.main.async {
                         let statusCode = (response as? HTTPURLResponse)?.statusCode ?? 0
                         var customError: AppError?
                         var receivedResponse: Any?
                         if let receivedData = data {
                             if responseType == .json {
                                 do {
-                                    // Ensure that the network data is available and that the byte count is greater that zero
-                                    if let jsonString = String.init(data: receivedData, encoding: String.Encoding.utf8)?.removingPercentEncoding,
+                                    // Ensure that the network data is available and
+                                    // that the byte count is greater that zero
+                                    if let jsonString = String.init(data:
+                                        receivedData, encoding:
+                                        String.Encoding.utf8)?.removingPercentEncoding,
                                         let jsonData = jsonString.data(using: .utf8) {
                                         // Perform the JSONSerialization into an array of Dictionaries
-                                        // Next use the convenience constructor in Article to create an Article object out of each object in the JSON array
-                                        let dictJson = try JSONSerialization.jsonObject(with: jsonData, options: .allowFragments)
-                                        receivedResponse = dictJson
+                                        // Next use the convenience constructor in Article to
+                                        // create an Article object out of each object in the JSON array
```

Comment:
I think there's a misconception of what is an `Article` and what is a `Result`. What you are getting is a result/response which contains an array of `Article`

Reply:
Its Done.